### PR TITLE
[OSDOCS-5880] Example `aws sts get-caller-identity` output is outdated

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-5880
Link to docs preview:

https://70679--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-config-aws-account
QE review:
- [x] QE has approved this change.

Additional information:

REVIEWERS: I found another example of this command in the current docs here: 
https://docs.openshift.com/rosa/rosa_getting_started/rosa-quickstart-guide-ui.html 
image
and used that example to correct the reported bug. 

image

### CODE REVIEW

File: modules/rosa-configuring-aws-account.adoc [hunk 1]
```diff
@@ -64,14 +64,7 @@ $ aws sts get-caller-identity
...
 ----
----------------------------------------------------------------------------------
-|                                GetCallerIdentity                              |
-+-------------------------------------------------------------------------------+
-|+-----------------------------------+-----------------------+-----------------+|
-||      Account        |                Arn              |       UserID        ||
-|+-----------------------------------+-----------------------+-----------------+|
-||  <account_name>     |  arn:aws:iam<string>:user:name  |      <userID>       ||
-|+-----------------------------------+-----------------------+-----------------+|
+<aws_account_id>    arn:aws:iam::<aws_account_id>:user/<username>  <aws_user_id>
```
Comment (xueli181114) [inline modules/rosa-configuring-aws-account.adoc:67]: Once customer set the output to table format the table still work. May I have more information why it is out of date?
`lixue@Xue-Lis-MacBook-Pro ~ % aws sts get-caller-identity
-----------------------------------------------------------------------------------
|                                GetCallerIdentity                                |
+--------------+----------------------------------------+-------------------------+
|    Account   |                  Arn                   |         UserId          |
+--------------+----------------------------------------+-------------------------+
|  301721915996|  arn:aws:iam::301721915996:user/xueli  |  AIDAJ6DYXER77CARDORK4  |
+--------------+----------------------------------------+-------------------------+`

**Comment by mletalie** [comment]
Hello @xueli181114, 
    I have reached out to the author of the ticket to respond to your comment/question. Given your remarks, does that also mean the other example where this command output is also incorrect? Please let me know, thanks!  
image

**Comment by xueli181114** [comment]
@mletalie How the command show the output depends on customers' local aws config settings.
If `table` configured like below The it will output a table
`lixue@Xue-Lis-MacBook-Pro ~ % cat ~/.aws/config 
[default]
output = table
region = us-west-2`

Otherwise, if configured with output=text, then it will output like text
`lixue@Xue-Lis-MacBook-Pro ~ % cat ~/.aws/config          
[default]
output = text
region = us-west-2`

You can see I am running same command, but get different output format due to different awscli configuration.

image

We cannot make sure how customers configured their local awscli. So I would suggest, we provide a command which can override the configuration so we can give the accurate output format. Like
image

**Comment by mletalie** [comment]
Hello @xueli181114,
   Had a chance to look at your comment in more depth; from what I can tell, the suggestion is just to add` --output text  ` to the current command. However, the other issue (and the original reason for this ticket) was to change the example output. Given that, do you think we need to go back to the original (current) example output but update the command as you suggested? Hope the question makes sense, please advise when you get a moment. Thanks.

**Comment by xueli181114** [comment]
Good approach! LGTM

File: modules/rosa-getting-started-install-configure-cli-tools.adoc [hunk 2]
```diff
@@ -50,7 +50,7 @@ You can alternatively use the `AWS_DEFAULT_REGION` environment variable to set t
 +
 [source,terminal]
 ----
-$ aws sts get-caller-identity
+$ aws sts get-caller-identity  --output text
```
Comment (bergerhoffer) [inline modules/rosa-getting-started-install-configure-cli-tools.adoc:53]: There are two spaces before the flag here. Seeing as this won't affect the command though, I won't hold up the merge over it.

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging (one super minor comment that won't hold up the merge, but feel free to fix it in a future PR if you care to).

**Pull request merged**